From 847eecbaa758b4e84c2dfc23421f3f415d8db817 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Tue, 27 Feb 2018 15:34:07 +0000
Subject: [PATCH] OS-6696 need iasl in tools (fix build) Reviewed by: Patrick
 Mooney <patrick.mooney@joyent.com> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 usr/src/cmd/acpi/iasl/Makefile.iasl | 2 --
 1 file changed, 2 deletions(-)

diff --git a/usr/src/cmd/acpi/iasl/Makefile.iasl b/usr/src/cmd/acpi/iasl/Makefile.iasl
index c4a68e23f7..167b85eabe 100644
--- a/usr/src/cmd/acpi/iasl/Makefile.iasl
+++ b/usr/src/cmd/acpi/iasl/Makefile.iasl
@@ -123,8 +123,6 @@ LFLAGS = -i -s
 
 .KEEP_STATE:
 
-.PARALLEL: $(OBJS) $(INTERMEDIATES)
-
 all: $(YACC_FILES) .WAIT $(PROG)
 
 $(IASL_OBJ_DIR)/aslcompiler.y: $(IASL_SRC_DIR)/aslparser.y
-- 
2.21.0

